model/bindings.go

```diff
@@ -0,0 +1,122 @@
+package model
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	yaml "github.com/cloudfoundry-incubator/candiedyaml"
+	"github.com/docker/libcompose/config"
+	"github.com/docker/libcompose/utils"
+	"github.com/rancher/rancher-compose/preprocess"
+	"github.com/rancher/rancher-compose/rancher"
+	"io/ioutil"
+	"os"
+	"reflect"
+)
+
+type mapLabel map[string]string
+
+//BindingProperty holds bindings
+type BindingProperty struct {
+	Binding
+}
+
```

Comment:
Actually I had another key within the BindingProperty initially that I removed. But once v2 is added there will also be a key for volumes, so I thought later I can add 2 keys in this outer struct, services and volumes. Does this seem okay or should I change it?